huaweicloud/services/dc/resource_huaweicloud_dc_virtual_interface_accepter.go

```diff
@@ -0,0 +1,134 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product DC
+// ---------------------------------------------------------------
+
+package dc
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/dc/v3/interfaces"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// @API DC GET /v3/{project_id}/dcaas/virtual-interfaces/{interfaceId}
+// @API DC PUT /v3/{project_id}/dcaas/virtual-interfaces/{interfaceId}
```

Comment:
```suggestion
// @API DC GET /v3/{project_id}/dcaas/virtual-interfaces/{virtual_interface_id}
// @API DC PUT /v3/{project_id}/dcaas/virtual-interfaces/{virtual_interface_id}
```
https://support.huaweicloud.com/api-dc/UpdateVirtualInterface.html

Reply:
Keep it consistent with other DC resources and do not modify it yet.